numba/unicode.py

```diff
@@ -1286,6 +1288,85 @@ def impl(a):
     return impl
 
 
+@register_jitable
+def _handle_capital_sigma(data, length, idx):
+    """
+    https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Objects/unicodeobject.c#L9856-L9883
```

Comment:
I don't think these URLs should go into docstrings without at least a prelude like "This is a translation of the function that handles the capital sigma character cf. <URL>". This comment applies to all docstrings that are of this form. Also, the reference to the CPython impl should go into a comment.

Reply:
I will add such descriptions.